tests/e2e/e2e_workflow.spec.js

```diff
@@ -66,14 +82,34 @@ describe('Basic end-to-end Workflow', function () {
     // stuck to the first build done
     sh.exec(`${ngBin} build -prod`);
     expect(existsSync(path.join(process.cwd(), 'dist'))).to.be.equal(true);
-    var mainBundlePath = path.join(process.cwd(), 'dist', 'main.js');
-    var mainBundleContent = fs.readFileSync(mainBundlePath, { encoding: 'utf8' });
-    // production: true minimized turns into production:!0
-    expect(mainBundleContent).to.include('production:!0');
+    if (!isMobileTest()) {
+      var mainBundlePath = path.join(process.cwd(), 'dist', 'main.js');
+      var mainBundleContent = fs.readFileSync(mainBundlePath, { encoding: 'utf8' });
+      // production: true minimized turns into production:!0
+      expect(mainBundleContent).to.include('production:!0');
+    }
+
     // Also does not create new things in GIT.
     expect(sh.exec('git status --porcelain').output).to.be.equal(undefined);
   });
 
+  if (isMobileTest()) {
+    it('Enables mobile-specific production features', () => {
```

Comment:
Or `it_not_mobile`? It would at least make the delta smaller, and get rid of awkward `if` blocks.